torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -202,19 +214,20 @@ class BatchDataBuffer {
   /// count of total example stored in the queue
   size_t total_example_count_in_queue_ = 0;
 
-  /// struct that contains a raw unwrapped batch unit. An unwrapped batch unit is
-  /// the raw data without 'optional' wrapper. It can be a collection of images,
-  /// utterances, e.t.c.
+  /// struct that contains a raw unwrapped batch unit. An unwrapped batch unit
+  /// is the raw data without 'optional' wrapper. It can be a collection of
+  /// images, utterances, e.t.c.
```

Comment:
Please avoid small formatting changes like this. They greatly increase the size of the PR, which means it takes 2x longer for me to review it, because I need to see every single changed line... If you really dislike your previous formatting, please submit it as a separate patch...